File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.cs
public class Solution {
    public bool CheckIfPangram(string sentence) {
        HashSet<char> hashh=new HashSet<char>();
        for(int i=0;i<sentence.Length;i++){
            hashh.Add(sentence[i]);
        }
        if(hashh.Count==26){
            return true;
        }
        else{
            return false;
        }

    }
}

File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    sentc=new Set();
    for(i=0;i<sentence.length;i++){
        sentc.add(sentence[i])
    }
    if(sentc.size===26){
        return true
    }
    else{
        return false
    }
};